Makefile

```diff
@@ -580,7 +582,7 @@ ci.e2e.build.%:
 e2e.build.centos-7: centos7 docker.clean-latest-ami
 
 # Run os-packages-artifacts outside devkit container.
-e2e.build.centos-7-offline: centos7-offline docker.clean-latest-ami
+e2e.build.centos-7-offline: centos7-offline docker.clean-latest-ami infra.aws.destroy
 
 e2e.build.rhel-7.9-offline-fips: rhel79-fips-offline docker.clean-latest-ami
```

Comment:
We also want to make a similar change for the `rhel-x.x-offline-fips` jobs, is that going to be done as part of this?

Reply:
Added! Just wanted to get this working with one OS first before delving in to more :)